### PR TITLE
News post for playtest-20170722

### PR DESCRIPTION
**Pull request by pchote**
I'm putting up the main post now for feedback / proof reading.
I would like to aim for an early saturday afternoon release, before the player count picks up.

Still TODO:
 - [x] Add Battlefield News entry
 - [x] Update the Download page generation to support the new macOS dmg packaging
 - [x] Update the Games page to support the new `openra-<mod>-<tag>://` url schemes
 - [x] Update the changelog hash in the news post
 - [x] Add release IDs after tagging the final builds.

image

### CODE REVIEW

**Comment by pchote** [comment]
I just noticed that we used a similar d2k campaign screenshot for the last release newspost.  I'll replace this with a screenshot of the new map editor features in d2k.

**Comment by pchote** [comment]
Updated d2k screenshot and added a Battlefield News entry:

screen shot 2017-07-20 at 22 48 04

File: content/news/playtest-20170722.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Don't forget, if you're on Windows or macOS you can have playtest-20170722 and release-20170527 installed at the same time, and switch between the two when joining servers with other players.  So head over to our [download page](/download/) and give it a try!
+
+<div class="about-todo-divider"></div>
+
+And now for a special note to our modding community:
+
+As OpenRA modding grew in popularity it became clear that requiring players to manually install mods *on top of* our OpenRA releases, with the in-game mod chooser to switch between mods, was more of a burden than a benefit.  We have listened to the feedback from both players and modders, and have spent the last few months working on a comprehensive solution.
+
+We have done this by decoupling the game engine (OpenRA.Game.exe and support files) from our default mods, and have put *the mod* in charge.  Mods now manage their own engine files, which means that they can exist *along side* our official mods with no dependencies between them.
```
Comment (obrakmann) [inline content/news/playtest-20170722.md]: typo: alongside is one word

File: content/games.html [hunk 2]
```diff
@@ -104,16 +104,20 @@
...
-        if (game['state'] == '1')
-          row.append('<td>'+game['location']+'<br /><span class="gameSubline"><a href="openra://'+game['address']+'">'+game['address']+'</a></span></td>');
-        else
+        if (game['state'] == '1') {
+          var handler = 'openra-' + modver[0] + '-' + modver[1] + '://';
+          if (modver[1] == 'release-20170527')
+            handler = 'openra://';
+
+          row.append('<td>'+game['location']+'<br /><span class="gameSubline"><a href="'+handler+game['address']+'">'+game['address']+'</a></span></td>');
+        } else
```
Comment (obrakmann) [inline content/games.html]: Not gonna block this, just a suggestion: would it be better to make the server name the link, and not the address? I have a feeling that many people miss that some IPs are links.

Comment (pchote) [inline content/games.html]: This would be best done in its own PR.

**Comment by pchote** [comment]
Updated changelog link and download ID.
